### ISSUE TITLE
missing formats due to auf1 API change

### ISSUE THREAD

**Issue by BergbauerSepp** [flashdagger/ytdlp-plugins#7]
Hello :)

last successful download 06.sep.2023

```
Linux XXXXXXXXX 5.10.0-23-amd64 #1 SMP Debian 5.10.179-1 (2023-05-12) x86_64 GNU/Linux

ytdlp-plugins -v https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023
[debug] Command-line config: ['-v', 'https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023']
[debug] ytdlp-plugins (v2023.01.11) loaded 10 plugins which are not part of yt-dlp. Use at your own risk.
[debug]  [auf1:radio]       via 'yt_dlp_plugins.extractor.auf1.Auf1RadioIE'          (v2023.07.10)
[debug]  [auf1]             via 'yt_dlp_plugins.extractor.auf1.Auf1IE'               (v2023.07.10)
[debug]  [d.tube:query]     via 'yt_dlp_plugins.extractor.dtube.DTubeQueryPluginIE'  (v2023.06.03)
[debug]  [d.tube:search]    via 'yt_dlp_plugins.extractor.dtube.DTubeSearchPluginIE' (v2023.06.03)
[debug]  [d.tube:user]      via 'yt_dlp_plugins.extractor.dtube.DTubeUserPluginIE'   (v2023.06.03)
[debug]  [d.tube]           via 'yt_dlp_plugins.extractor.dtube.DTubePluginIE'       (v2023.06.03)
[debug]  [pm-wissen:search] via 'yt_dlp_plugins.extractor.servustv.PmWissenSearchIE' (v2023.01.08)
[debug]  [pm-wissen]        via 'yt_dlp_plugins.extractor.servustv.PmWissenIE'       (v2023.01.08)
[debug]  [servustv:search]  via 'yt_dlp_plugins.extractor.servustv.ServusSearchIE'   (v2023.01.08)
[debug]  [servustv]         via 'yt_dlp_plugins.extractor.servustv.ServusTVIE'       (v2023.01.08)
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.9.2 (CPython x86_64 64bit) - Linux-5.10.0-23-amd64-x86_64-with-glibc2.31 (OpenSSL 1.1.1n  15 Mar 2022, glibc 2.31)
[debug] exe versions: ffmpeg 4.3.6-0, ffprobe 4.3.6-0, phantomjs broken, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.14.1, brotli-1.0.9, certifi-2020.06.20, mutagen-1.45.1, pyxattr-0.7.2, sqlite3-2.6.0, websockets-10.2
[debug] Proxy map: {}
[debug] Extractor Plugins: Auf1IE, Auf1RadioIE, DTubePluginIE, DTubeQueryPluginIE, DTubeSearchPluginIE, DTubeUserPluginIE, PmWissenIE, PmWissenSearchIE, ServusSearchIE, ServusTVIE
[debug] Plugin directories: ['/usr/local/lib/python3.9/dist-packages/yt_dlp_plugins']
[debug] Loaded 1865 extractors
[auf1] Extracting URL: https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023
[auf1] nachrichten-auf1-vom-14-september-2023: Downloading JSON metadata
ERROR: [auf1] nachrichten-auf1-vom-14-september-2023: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 1560, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 1709, in __extract_info
    return self.process_ie_result(ie_result, download, extra_info)
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 1768, in process_ie_result
    ie_result = self.process_video_result(ie_result, download=download)
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 2732, in process_video_result
    self.raise_no_formats(info_dict)
  File "/usr/local/lib/python3.9/dist-packages/yt_dlp/YoutubeDL.py", line 1073, in raise_no_formats
    raise ExtractorError(msg, video_id=info['id'], ie=info['extractor'],
yt_dlp.utils.ExtractorError: [auf1] nachrichten-auf1-vom-14-september-2023: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

**Comment by flashdagger** [comment]
Hi @BergbauerSepp, I'm happy to see the plugin is still in use. Auf1 is changing their API from time to time and made a big overhaul recently. I already looked into it and will work on a new release.

P.S.: Did you know you can now call `yt-dlp <Auf1-URL>` since yt-dlp now has a native plugin system, making ytdlp-plugins actually obsolete?

**Comment by BergbauerSepp** [comment]
Hello @flashdagger no, I didn't know that. I just tested it, yt-dlp is also broken. I will also make a report there. Thank you for developing this plugin, it has served me well. If you develop it further, I will continue to use yours.

**Comment by flashdagger** [comment]
You don't need to. Auf1-URLs are supported only by the ytdlp-auf1 plugin by now. Since version 2023.01.06 yt-dlp also supports plugins from python packages, so you don't need to call them via `ytdlp-plugins`.
Just wait for the fix in the next release.

**Comment by BergbauerSepp** [comment]
ah now I understand😅 I was wondering why your extractors were used when calling yt-dlp <AUF1-url>. I'll wait, thanks a lot

**Comment by flashdagger** [comment]
New release `2023.9.15` is available. You can install it with:
```
python -m pip install -U ytdlp-auf1
```

**Comment by BergbauerSepp** [comment]
Thank you very much :) works again, you're great!


```
 yt-dlp -vU https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023
[debug] Command-line config: ['-vU', '--verbose', 'https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (pip)
[debug] Python 3.9.2 (CPython x86_64 64bit) - Linux-5.10.0-23-amd64-x86_64-with-glibc2.31 (OpenSSL 1.1.1n  15 Mar 2022, glibc 2.31)
[debug] exe versions: ffmpeg 4.3.6-0, ffprobe 4.3.6-0, phantomjs broken, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.14.1, brotli-1.0.9, certifi-2020.06.20, mutagen-1.45.1, pyxattr-0.7.2, sqlite3-2.6.0, websockets-10.2
[debug] Proxy map: {}
[debug] Extractor Plugins: Auf1IE, Auf1RadioIE, PmWissenIE, PmWissenSearchIE, ServusSearchIE, ServusTVIE
[debug] Plugin directories: ['/usr/local/lib/python3.9/dist-packages/yt_dlp_plugins']
[debug] Loaded 1861 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
yt-dlp is up to date (stable@2023.07.06)
[auf1] Extracting URL: https://auf1.tv/nachrichten-auf1/nachrichten-auf1-vom-14-september-2023
[auf1] nachrichten-auf1-vom-14-september-2023: Downloading JSON metadata
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, size, br, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] nachrichten-auf1-vom-14-september-2023: Downloading 1 format(s): 1080p
[debug] Invoking http downloader on "https://gegenstimme.tv/static/webseed/6ab80d27-86ed-42af-9ca3-8a1216857817-1080.mp4"
[download] Destination: Nachrichten AUF1 vom 14. September 2023 [nachrichten-auf1-vom-14-september-2023].mp4
[download]   1.4% of  294.85MiB at    8.21MiB/s ETA 00:35^C
ERROR: Interrupted by user
```
